src/implementations/vanilla/components/text-area/package.json

```diff
@@ -7,7 +7,6 @@
   "style": "text-area.scss",
   "sass": "text-area.scss",
   "dependencies": {
-    "@ecl/theme-dev": "3.11.0",
     "@ecl/vanilla-layout-grid": "3.11.0"
```

Comment:
we are now using the new @ecl/vanilla-layout-form, should be listed here as well. About the new package maybe we can remove the mixins folder and access it simply as @ecl/vanilla-layout-form/mixins